crawl-ref/source/dat/des/branches/snake.des

```diff
@@ -1398,12 +1398,13 @@ MONS:   naga / nothing
 MONS:   naga warrior / naga sharpshooter w:5 / naga w:15 / nothing
 MONS:   naga mage w:20 / naga ritualist / naga sharpshooter / naga w:30 / \
         nothing w:20
-MONS:   guardian serpent w:20 / salamander mystic w:5 / naga ritualist w:5 / \
+MONS:   guardian serpent / salamander mystic w:5 / naga ritualist w:5 / \
         nothing
 MONS:   greater naga w:30 / naga warrior w:5 / naga sharpshooter w:5
-MONS:   anaconda / black mamba / shock serpent w:5 / mana viper w:15
+MONS:   anaconda / black mamba / shock serpent w:5 / mana viper
 
-KMONS:  Z = Vashnia / greater naga, greater naga
+KMONS:  Z = Vashnia / greater naga
```

Comment:
This means if Vashnia was placed already it won't even put a greater naga; is that intentional?